core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -452,6 +555,10 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
       case (expected: SparkListenerStageCompleted, actual: SparkListenerStageCompleted) =>
         // accumulables can be different, so only check the stage Id
         assert(expected.stageInfo.stageId == actual.stageInfo.stageId)
+      case (expected: SparkListenerTaskEnd, actual: SparkListenerTaskEnd) =>
+        // taskInfo, taskExecutorMetrics, and taskMetrics will be different object references,
```

Comment:
I'll change the == to === in all the cases here.
I didn't want to bother checking the contents of the objects because they are not critical or central to what we're verifying. It's just an additional complication. Same reason we do not check the values of the accumulables.

Reply:
ok on not checking them -- but then that comment is misleading, I'd delete it.

Reply:
Ok, deleted the comment.